modules/authentication.py

```diff
@@ -169,11 +170,11 @@ def verify_response(
             session['macaroon_discharge']
```

Comment:
The docstring above needs updating to reflect the new behaviour of this function